_example-projects/search/index.md

```diff
@@ -1,7 +1,7 @@
 ---
 title: Knowledge Supported Search 
 subheadline: Example project
-permalink: /use-case-gallery/search/
+permalink: /use-case-gallery/example-projects/search/
```

Comment:
It's strange that examples are under the use-case-gallery in the URL - why not `/use-case-gallery/...` and `/example-projects/...`?

Reply:
Because they are under one category (examples menu item) which also has an overview text.